runtime/reference/node.md

```diff
@@ -273,7 +330,7 @@ A full list of all supported linting rules can be found on
 [https://lint.deno.land/](https://lint.deno.land/). To learn more about how to
 configure the linter, check out the [`deno lint` subcommand](../tools/linter/).
 
-#### Formatting (optional)
+#### Formatting
```

Comment:
probably should mention how this relates to prettier here? IE "instead of adding prettier to your devDependencies, you can instead use Deno's built-in zero-config code formatter `deno fmt`"